clients/adapter-kafka/src/main/java/org/eclipse/hono/adapter/client/command/kafka/KafkaBasedInternalCommandConsumer.java

```diff
@@ -129,8 +134,25 @@ public KafkaBasedInternalCommandConsumer(
         consumerCreator = () -> consumer;
     }
 
+    /**
+     * Sets the vert.x context to run on.
+     * <p>
+     * If not set explicitly, the context is determined during startup.
+     *
+     * @param context The context.
+     */
+    void setContext(final Context context) {
```

Comment:
I see that this method is used only for unit tests. Can't we just add the `context` as an parameter to the constructor with package level visibility, which is only being used by the tests. So that the tests need not invoke this method additionally.

Reply:
Good point, I've changed the commit accordingly.